src/marqo/tensor_search/on_start_script.py

```diff
@@ -65,6 +65,7 @@ def run(self):
                 "ensure that Marqo is configured properly for this. See "
                 f"{marqo_docs.configuring_marqo()} for more details. Error: {e}"
             )
+            raise e
```

Comment:
This change is requested by Cloud team. If anything goes wrong during bootstrapping, Marqo instance will be shutdown. This guarantees that Marqo will always run on a properly bootstrapped Vespa application. Marqo Cloud can leverage this to capture and manage upgrade failure properly as well.

Reply:
Can we change the wording of the error to "Failed to bootstrap Marqo"?

Reply:
It makes sense to me to mention vector store here since we are bootstrapping the vector store to work with current version of Marqo.